scripts/versionbump.js

```diff
@@ -0,0 +1,21 @@
+const replace = require('replace-in-file');
+const pjson = require(`${process.cwd()}/package.json`);
+
+const files = process.argv.slice(2);
+if (files.length == 0) {
+  console.error('Please provide files to bump');
+  process.exit(1);
+}
+
+replace({
+  files: files,
+  from: /\d+\.\d+.\d+(?:-\w+\.\w+)?/g,
```

Comment:
Fragile, but I don't think it needs to be more complex than this.

Also regexp should be `\d+\.\d+.\d+(?:-\w+(?:\.\w+)?)?`, as 2nd part of a group is optional.

Before:
![screen shot 2018-06-28 at 11 06 36](https://user-images.githubusercontent.com/1523305/42024833-5d5719c8-7ac3-11e8-9a0d-d21302b3d7ac.png)

After:
![screen shot 2018-06-28 at 11 06 28](https://user-images.githubusercontent.com/1523305/42024834-5d7468a2-7ac3-11e8-8784-fe3f04a4afee.png)